src/projects/apps/designer/typography.md

```diff
@@ -0,0 +1,24 @@
+# Typography
+
+These properties cover the font and it's styling used with this component.
+
+### Font Family
+Select a set of fonts to use in priority order (ie the first font will be the font that will be used unless it cannot be loaded, in which case the browser will try to use the next and so on). A font is one of the following:
+* **Google font**: loaded from the Google fonts library.
```

Comment:
Would probably be good to add a link to the google font library

Reply:
Yes, agreed

Reply:
done